ospf6d/ospf6_main.c

```diff
@@ -214,9 +245,15 @@ int main(int argc, char *argv[], char *envp[])
 	access_list_init();
 	prefix_list_init();
 
+	/* Configuration processing callback initialization. */
+	cmd_init_config_callbacks(ospf6_config_start, ospf6_config_end);
+
 	/* initialize ospf6 */
 	ospf6_init(master);
 
+	/* Configuration processing callback initialization. */
+	cmd_init_config_callbacks(ospf6_config_start, ospf6_config_end);
```

Comment:
Btw, do we need to call callbacks twice?

Reply:
Nope - let me fix this and squash the commit.

Reply:
Fixed and pushed squashed commit after rerunning ospfv3_basic_functionality tests.